### PR TITLE
Add a recipe for dbus

### PR DESCRIPTION
**Pull request by ccordoba12**
This is the first dependency for Qt5 on Linux :-)

### CODE REVIEW

**Comment by ccordoba12** [comment]
@ocefpaf, this should be ready too. We are not creating packages for OS X because we don't need them right now :-)

**Comment by ocefpaf** [comment]
>  We are not creating packages for OS X because we don't need them right now :-)

Fair enough but it would be nice to known if just enabling `OS X` "works" or if we need to open an issue with more information on what may be broken there.

File: recipes/dbus/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+CPPFLAGS=-I${PREFIX}/include \
+LDFLAGS=-L${PREFIX}/lib      \
+  ./configure --prefix=${PREFIX} \
+              --with-launchd-agent-dir=${PREFIX} \
+              --disable-systemd \
+              --disable-selinux
+make
+make install
```
Comment (ocefpaf) [inline recipes/dbus/build.sh]: `make check` ?

File: recipes/dbus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - expat
+
+test:
+  commands:
+    - dbus-daemon --version
+    - dbus-uuidgen
+
+about:
+  summary: "D-Bus is a message bus system, a simple way for applications to talk to one another. "
+  license: GPL
```
Comment (ocefpaf) [inline recipes/dbus/meta.yaml]: Version?

**Comment by ccordoba12** [comment]
Ok, I'll activate `osx` to see what happens.

**Comment by ccordoba12** [comment]
Just for the record, this got compiled for OS X too at the end :-)

**Comment by ocefpaf** [comment]
>  this got compiled for OS X too at the end :-)

:tada: 

Thanks @ccordoba12!

**Comment by jakirkham** [comment]
cc @asmeurer

**Comment by asmeurer** [comment]
Thanks. Dbus is needed for [emacs](https://github.com/conda-forge/staged-recipes/pull/1165) on OS X.

**Pull request merged**